pkgs/development/python-modules/imageio/default.nix

```diff
@@ -35,7 +35,7 @@ buildPythonPackage rec {
     export IMAGEIO_USERDIR="$TMP"
     export IMAGEIO_NO_INTERNET="true"
     export HOME="$(mktemp -d)"
-    py.test
+    py.test -k "not test_get_exe_env"
```

Comment:
(optional) This can be using `pytestCheckHook`

Reply:
thanks! neat